Build-exe-and-msi.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3
+"""Build to executables and MSI installer using py2exe and other tools."""
+import os
+import pathlib
+import platform
+import re
+import shutil
+import sys
+from os.path import exists, isdir, join
+from tempfile import gettempdir
+
+from lxml import etree
+from py2exe import freeze
+
+from config import appcmdname, appname, appversion, appversion_nobuild, copyright, git_shorthash_from_head
+from constants import GITVERSION_FILE
+
+###########################################################################
+# Check we're on a supported platform
+###########################################################################
+if sys.version_info[0:2] != (3, 10):
+    raise AssertionError(f'Unexpected python version {sys.version}')
+
+if sys.platform == 'win32':
+    assert platform.architecture()[0] == '32bit', 'Assumes a Python built for 32bit'
+    import py2exe  # noqa: F401 # Yes, this *is* used
+    dist_dir = 'dist.win32'
+
+else:
+    assert False, f'Unsupported platform {sys.platform}'
```

Comment:
May as well just `raise AssertionError()` here if you want specifically that exception

Reply:
Changes to use that.